### PR TITLE
IBM Cloud docs: add apply step to kubeflow-gateway for https config

### PR DESCRIPTION
**Pull request by marchmallow**

### CODE REVIEW

File: content/en/docs/ibm/deploy/authentication.md [hunk 1]
```diff
@@ -135,6 +135,10 @@ of the column `Hostname` in step 4:
           privateKey: /etc/istio/ingressgateway-certs/tls.key
           serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
     ```
+    
+    ```shell
+    kubectl apply -f kubeflow-gateway.yaml -n kubeflow
+    ```
```
Comment (shawnzhu) [inline content/en/docs/ibm/deploy/authentication.md:141]: Thanks!

Do you mind adding some explanation saying "Run this command to apply changes to the gateway `kubeflow-gateway`:"

Comment (marchmallow) [reply content/en/docs/ibm/deploy/authentication.md:141]: sure will do

Comment (marchmallow) [reply content/en/docs/ibm/deploy/authentication.md:142]: @shawnzhu updated .. see if looks good now?

File: content/en/docs/ibm/deploy/authentication.md [hunk 2]
```diff
@@ -135,12 +135,13 @@ of the column `Hostname` in step 4:
           privateKey: /etc/istio/ingressgateway-certs/tls.key
           serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
     ```
-    
+
+8. Execute the following command to apply the changes to the gateway `kubeflow-gateway`:
```
Comment (8bitmp3) [inline content/en/docs/ibm/deploy/authentication.md:139]: Nit: add a blank line before the code block

```suggestion
8. Execute the following command to apply the changes to the gateway `kubeflow-gateway`:

```

**Comment by 8bitmp3** [comment]
Hi @marchmallow I think you should also sign the Contributor License Agreement (CLA) in order for the PR to get OKed by the system even after @Tomcli and @shawnzhu approve your changes. Check the first `google-cla` message above for details including which email address to use (for corporate/individual users), etc 👍.

**Comment by marchmallow** [comment]
@googlebot I signed it!

**Comment by Tomcli** [comment]
@marchmallow are you able to sign the CLA?

**Pull request closed without merging**